src/main.rs

```diff
@@ -196,6 +196,11 @@ Must be used with --hub-token or --hub-token-file")
             .help("Location of file with token that can be used for accessing the Docker Hub, used when proxying Docker Hub images")
             .takes_value(true)
         )
+        .arg(
+            Arg::with_name("enable-cors")
+                .long("enable-cors")
+                .help("Enable Cross-Origin Resource Sharing(CORS) requests. Default: false")
```

Comment:
Can you remove the "default false" bit? Also add something like "Used to allow access from web apps (e.g. GUIs).", so that people understand why it's there.

Reply:
Will do.